Source/WebKit/WebKitSwift/TextExtraction/WKTextExtractionItem.swift

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2024 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+// BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+// THE POSSIBILITY OF SUCH DAMAGE.
+
+import CoreFoundation
+import Foundation
+
+@available(iOS 17.0, macOS 12.0, *)
+@objc(WKTextExtractionItem) public class WKTextExtractionItem: NSObject {
```

Comment:
I think this can just be `@objc` (you only need the label if you want the objc class to be named different than the swift class)

Reply:
I think this caused soft-linking the ObjC class to fail when I omitted it, for some reason. I can try again and confirm..

Reply:
Yeah, `@objc` doesn't work:

```
/Users/whsieh/Build/Release/usr/local/include/WebKitSwift/WKTextExtractionItem.h:43:12: error: declaration has external linkage, but dynamic library doesn't have symbol 'WKTextExtractionItem'
   43 | @interface WKTextExtractionItem : NSObject
…
```

Additionally, removing the declarations results in `getWKTextExtractionItemClass()` returning `nil`.